src/components/Select/Select.tsx

```diff
@@ -57,8 +57,14 @@ export type Props = {
   isLoading?: boolean;
   /** if options list is virtualized */
   isVirtualized?: boolean;
-  /** A callback that's called when the user clicks the 'clear' icon */
+  /** A callback that's called when the user clicks the 'clear' icon of the (Single) Select, or the 'clear all' button of the MultiSelect */
   onClear?: () => void;
+  /** A callback that's called when the user clicks the 'clear' icon of a specific Chip in MultiSelect */
+  onOptionDelete?: (option: SelectOption) => void;
+  /** If true the user can select multiple options */
+  multi?: boolean;
+  /** The selected options in case of multiSelect */
+  selectedOptions?: SelectOption[];
```

Comment:
Can we add a @TODO here for v5 to merge selectedOptions with selectedOption? Value maybe []